src/OpenBinary/reader.jl

```diff
@@ -133,18 +125,17 @@ function PSRI.open(
         _first_stage = read(ioh, Int32) #month or week
         first_year = read(ioh, Int32) # year
         #
-        unit_buffer = Vector{Cchar}(undef, 7)
-        read!(ioh, unit_buffer)
-        unit_str = strip(join(unit_buffer))
+        unit_str = strip(join(Char.(read!(ioh, Vector{Cchar}(undef, 7)))))
         #
         name_length = read(ioh, Int32)
 
         ##
         variable_by_hour = 0
         number_blocks = Int32[]
     else
-        skip_store(ioh, 4, skips, store_skips)
-        skip_store(ioh, 4, skips, store_skips)
```

Comment:
You gave up on this?
Whats the reason?

Reply:
It was important when I was investigating that binary data alignment issue last year. Now that it is stable we don't need it anymore.